Classes/KIFUITestActor.m

```diff
@@ -387,24 +387,32 @@ - (void)enterTextIntoCurrentFirstResponder:(NSString *)text
 
 - (void)enterTextIntoCurrentFirstResponder:(NSString *)text fallbackView:(UIView *)fallbackView
 {
+    UIView *fallback = fallbackView;
+    if (!fallback) {
+        UIResponder *firstResponder = [[[UIApplication sharedApplication] keyWindow] firstResponder];
+
+        if ([firstResponder isKindOfClass:[UIView class]]) {
+            fallback = (UIView *)firstResponder;
+        }
+    }
+
+    id<UITextInputTraits> textInputField = nil;
```

Comment:
Unfortunately doesn't seem to work: https://travis-ci.org/kif-framework/KIF/jobs/229223229

```
    ✗ testEnteringEmojiCharactersIntoViewWithAccessibilityLabel, The step timed out after 10.00 seconds: Failed to get text " 😓He😤ll👿o" in field; instead, it was " 😓He😤lol👿o"
```